src/configuration/prima.rs

```diff
@@ -72,14 +72,26 @@ impl Configuration for PrimaConfiguration {
 
 /// Represent an environment in which the datadog client runs.
 /// This is useful for enforcing rules based on environment for every application that uses the library.
-#[derive(PartialEq, Eq, Debug)]
+#[derive(PartialEq, Eq, Debug, Clone, Copy)]
```

Comment:
I've seen some talk of avoiding Copy in most cases, since it changes the normal semantics of how we expect most values to work, and if we end up doing something that makes Environment heavy to copy later, it can't really be rolled back... I'm not totally against it here, but it's worth considering if it's strictly necessary (as you mention below, being explicit is often better)

Reply:
Okay no problem. I'll keep the Clone around though as we already implement that in the other enum.